brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-3336 Fix: Image gallery - Resolved an issue wherein the lightbox was not working correctly within container.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-3336

### Screenshots

- https://d.pr/v/01W83E

### Types of changes




- Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?

- Drag and drop 2 column container.
- Use Image gallery in one of the container.
- Select Click Event as lightbox.
- Publish the page and visit frontend.
- Click on any image from lightbox. make sure lightbox visible properly


### Checklist:
